Supertext.Base.Caching.Redis/RedisCache.cs

```diff
@@ -60,4 +84,9 @@ public async Task RemoveAsync(string key, CancellationToken token = default)
     {
         await _distributedCache.RemoveAsync(key, token);
     }
+
+    private static DistributedCacheEntryOptions CreateDistributedCacheEntryOptions(TimeSpan absoluteExpirationRelativeToNow)
```

Comment:
Wouldn't `lifetime` or `timeToLive` be shorter and just as easily understood? ICMP packets (which ping, the network command, uses) have a TTL (time-to-live) value, so the term is not new in computing.